bindata/oauth-apiserver/deploy.yaml

```diff
@@ -101,18 +101,36 @@ spec:
         - mountPath: /var/log/oauth-apiserver
           name: audit-dir
         livenessProbe:
-          initialDelaySeconds: 30
           httpGet:
             scheme: HTTPS
             port: 8443
             path: healthz
+          initialDelaySeconds: 0
+          periodSeconds: 10
+          timeoutSeconds: 1
+          successThreshold: 1
+          failureThreshold: 3
         readinessProbe:
-          failureThreshold: 10
           httpGet:
             scheme: HTTPS
             port: 8443
             path: readyz
-      terminationGracePeriodSeconds: 70 # a bit more than the 60 seconds timeout of non-long-running requests
+          initialDelaySeconds: 0
+          periodSeconds: 5
+          timeoutSeconds: 1
+          successThreshold: 1
+          failureThreshold: 1
```

Comment:
Why is readiness probed for double as often (every 5s vs. 10s) compared to livenessProbe?

In the same realm, why are we just trying once (failureThreshold) before marking Unready vs. three times compared to liveness?

Reply:
> Why is readiness probed for double as often (every 5s vs. 10s) compared to livenessProbe?

liveness should basically never go false, but readyz should go false every time a shutdown starts.  So we don't really need to check liveness often, but readiness we want to responsive to.

> In the same realm, why are we just trying once (failureThreshold) before marking Unready vs. three times compared to liveness?

if for some reason this pod cannot answer requests (ready=false), we want it out of the service LB quickly and it can re-enter.  But if liveness fails, we're going to kill the container, so we want to give it a little more grace.   The stakes are higher.